tests/rules/no-empty-blocks.test.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @fileoverview Tests for no-empty-blocks rule.
+ * @author Nicholas C. Zakas
+ */
+
+//------------------------------------------------------------------------------
+// Imports
+//------------------------------------------------------------------------------
+
+import rule from "../../src/rules/no-empty-blocks.js";
+import css from "../../src/index.js";
+import { RuleTester } from "eslint";
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+	plugins: {
+		css,
+	},
+	language: "css/css",
+});
+
+ruleTester.run("no-empty-blocks", rule, {
+	valid: ["a { color: red; }", "@media print { a { color: red; } }"],
+	invalid: [
+		{
+			code: "a { }",
+			errors: [
+				{
+					messageId: "emptyBlock",
+					line: 1,
+					column: 3,
+					endLine: 1,
+					endColumn: 6,
+				},
+			],
+		},
```

Comment:
```suggestion
		},
		{
			code: "a { /* comment */ }",
			errors: [
				{
					messageId: "emptyBlock",
					line: 1,
					column: 3,
					endLine: 1,
					endColumn: 20,
				},
			],
		},
```

It would be good to add a test with a comment inside.